api/src/org/labkey/api/webdav/AbstractWebdavResource.java

```diff
@@ -96,7 +96,7 @@ protected AbstractWebdavResource(Path folder, String name)
 
     protected AbstractWebdavResource(Path folder, String name, WebdavResolver resolver)
     {
-        super(folder, name, resolver);
+        super(folder.append(name), resolver);

```

Comment:
```suggestion
        super(folder, name, resolver);
```
Still not sure why this needs to change.  Am I looking at the wrong version of the file somehow?

Reply:
This was homeschooling fatigue. Intended to revert it.